### PR TITLE
style: added updated styling

### PR DESCRIPTION
**Pull request by andskaal**

### CODE REVIEW

**Comment by dusan-kovacik-profiq** [comment]
I'm not able to select custom UI. Also, why is the page content not expanding more?
image

**Comment by andskaal** [comment]
> I'm not able to select custom UI. Also, why is the page content not expanding more?
> image

i am facing an issue where the screen goes white after selecting the custom UI, and Hana will help me troubleshoot this. I am aware of it, and sorry for not making that clear in the beginning 😄

File: packages/web-ts/src/App.css [hunk 1]
```diff
@@ -1,13 +1,212 @@
...
+}
+
+.indykite-sdk-container .full-view-container .select-container .language-container .active-display .arrow{
+  position: absolute;
+  top: -1px;
+  right: 7.5px;
+}
+
+.indykite-sdk-container .full-view-container .select-container .language-container .active-display .arrow svg{
+  fill: #FFF;
```
Comment (hanny21) [inline packages/web-ts/src/App.css:100]: be consistent with the hex colors.. sometimes you are using lowercase and sometimes uppercase..

File: packages/web-ts/src/App.tsx [hunk 2]
```diff
@@ -51,11 +49,12 @@ const App: React.FC = () => {
         localeConfig: null,
       });
     }
+    setLangToggled(false)
   };
 
-  const handleOnUIUseChange = (e: any) => {
-    localStorage.setItem("whatUiToUse", e.target.name);
-    setState(e.target.name);
+  const handleOnUIUseChange = (type: any) => {
```
Comment (hanny21) [inline packages/web-ts/src/App.tsx:55]: type of the param is `string`, so rather use `string` instead of `any`

File: packages/web-ts/src/App.tsx [hunk 3]
```diff
@@ -77,73 +76,45 @@ const App: React.FC = () => {
...
+    <div className="indykite-sdk-container">
+      <div className="full-view-container">
+        <div className="select-container">
+          <div className="toggle-container">
+            <button onClick={() => handleOnUIUseChange("built-in")} className={`btn-option ${state === "built-in" ? "selected":""}`}>Built in</button>  
+            <button onClick={() => handleOnUIUseChange("custom")} className={`btn-option ${state === "custom" ? "selected":""}`}>Custom</button>
+          </div>
+          <div className="language-container" tabIndex={0} onBlur={() => setLangToggled(false)}>
+            <div onClick={() => setLangToggled(!langToggled)} className={`active-display ${langToggled && "toggled"}`}>
+              <span className="selected-label">{renderDisplayLanguage()}</span>
```
Comment (hanny21) [inline packages/web-ts/src/App.tsx:102]: would be good to be consistent how you assign conditional classNames.. looks weird if it is not done same in same file..

`{state === "built-in" ? "selected":""}`
`${langToggled && "toggled"}`

File: packages/web-ts/src/assets/arrow-down.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512"><!--! Font Awesome Pro 6.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2022 Fonticons, Inc. --><path d="M310.6 246.6l-127.1 128C176.4 380.9 168.2 384 160 384s-16.38-3.125-22.63-9.375l-127.1-128C.2244 237.5-2.516 223.7 2.438 211.8S19.07 192 32 192h255.1c12.94 0 24.62 7.781 29.58 19.75S319.8 237.5 310.6 246.6z"/></svg>
```
Comment (hanny21) [inline packages/web-ts/src/assets/arrow-down.svg:1]: linter error at the end of the file, please fix it

**Review by hanny21 — CHANGES_REQUESTED** [review]
Looks good :tada: , but found few issues and left few comments:

1. when I select the `CZ` as language and refresh the page, the page is loaded with `EN`. this bug is introduced in this PR, works fine in deployed version.. probably you removed too much code? :thinking: 

2. there are always vertical and horizontal scrollbars visible even if there is no overflow.. looks really bad

4. when I load page for the first time, the container does not have minimum width, is that desired?
image

File: packages/web-ts/package.json [hunk 5]
```diff
@@ -18,7 +18,9 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "fix:eslint": "eslint --fix \"src/**/*.{js,jsx}\"",
+    "fix:prettier": "prettier --loglevel warn --write ."
```
Comment (hanny21) [inline packages/web-ts/package.json:23]: the eslint script not working

**Review by hanny21 — APPROVED** [review]

**Review by dusan-kovacik-profiq — APPROVED** [review]

**Pull request merged**